src/kusto/azext_kusto/generated/_help.py

```diff
@@ -1,758 +1,798 @@
+               az kusto database show --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--resource-group "kustorptest"
+"""
+
+helps['kusto database create'] = """
+    type: command
+    short-summary: "Creates or updates a database."
+    parameters:
+      - name: --read-write-database
+        short-summary: "Class representing a read write database."
+        long-summary: |
+            Usage: --read-write-database soft-delete-period=XX hot-cache-period=XX location=XX kind=XX
+
+            soft-delete-period: The time the data should be kept before it stops being accessible to queries in \
+TimeSpan.
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+      - name: --read-only-following-database
+        short-summary: "Class representing a read only following database."
+        long-summary: |
+            Usage: --read-only-following-database hot-cache-period=XX location=XX kind=XX
+
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+    examples:
+      - name: KustoDatabasesCreateOrUpdate
+        text: |-
+               az kusto database create --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--parameters "{\\"location\\":\\"westus\\",\\"properties\\":{\\"softDeletePeriod\\":\\"P1D\\"}}" --resource-group \
+"kustorptest"
+"""
+
+helps['kusto database update'] = """
+    type: command
+    short-summary: "Updates a database."
+    parameters:
+      - name: --read-write-database
+        short-summary: "Class representing a read write database."
+        long-summary: |
+            Usage: --read-write-database soft-delete-period=XX hot-cache-period=XX location=XX kind=XX
+
+            soft-delete-period: The time the data should be kept before it stops being accessible to queries in \
+TimeSpan.
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+      - name: --read-only-following-database
+        short-summary: "Class representing a read only following database."
+        long-summary: |
+            Usage: --read-only-following-database hot-cache-period=XX location=XX kind=XX
+
+            hot-cache-period: The time the data should be kept in cache for fast queries in TimeSpan.
+            location: Resource location.
+            kind: Required. Kind of the database
+    examples:
+      - name: KustoDatabasesUpdate
+        text: |-
+               az kusto database update --cluster-name "kustoclusterrptest4" --database-name "KustoDatabase8" \
+--parameters "{\\"properties\\":{\\"softDeletePeriod\\":\\"P1D\\"}}" --resource-group "kustorptest"
```

Comment:
@astauben --parameters is an unrecognized argument.

Reply:
fixed. Thanks!